docs/Getting Started/Installation/index.md

```diff
@@ -189,3 +189,84 @@ from within the JavaEE directory and a JAR should be generated in the build/libs
+In a new directory, save your `SDL.min.js` build there, and then run the following: `npm init`. Press Enter repeatedly through the prompts to set up some default npm configuration. Then, install the `express` package by running `npm install express --save`. Express is a popular Node.js package that allows easy setup of a server. Finally, in a new `index.js` file, save the following to it:
+```js
+const express = require('express');
+const app = express();
+app.use(express.static(__dirname));
+const PORT = process.env.PORT || 3000;
+const server = app.listen(PORT, async function () {
+    console.log('Server running on port', PORT);
+});
+```
+This code will start up a server on port 3000 on localhost, and serve any files in the directory it is running in. Now that the server code is complete, create a new file named `index.html` that will contain the app logic and save it in the same directory. Make the contents of the HTML file the following:
+```html
+<html>
+    <head>
+        <script src='./SDL.min.js'></script>
+    </head>
+    <body>
+        <script>
+            console.log(SDL);
+        </script>
+    </body>
+</html>
+```
+Finally, run the server by running this in the same directory: `node index.js`. The console should print `Server running on port 3000`. Go to your browser and enter `localhost:3000` in the address bar. If you open up your browser's console on the blank page that shows up you should see the SDL library version be printed and the imported SDL object. If so, then you have successfully set up SDL in your browser! If not, then make sure your SDL build name is correct, and that the HTML file, build file, and JS server file are all in the same directory.
+
+## Node.js Setup
+This build allows you to create apps that run on your computer. [Node.js](https://nodejs.org/en/) and npm are the tools you will need to integrate this library into what will be your app.
+In a new directory, save your `SDL.min.js` build there, and then create a new file in the same directory named `index.js`. Make the contents of that file the following:
+```js
+const SDL = require('./SDL.min.js');
+console.log(SDL);
+```
+Finally, run the file: `node index.js`. You should see the SDL library version be printed and the imported SDL object. If so, then you have successfully set up the SDL library!
+
+## WebEngine App Setup
+WebEngine apps use the vanilla JavaScript build, and are set up in a similar fashion to JS apps where it will also run in the browser. Set up the `index.js` and `index.html` file like in the **Vanilla JavaScript Setup** section. The majority of the configuration for the app will now be separated into a `manifest.js` file and then imported into the `index.html` file. Create a `manifest.js` file like below and save it in the same directory as the other two files. The entrypoint's value should have the same name as your app's HTML file. 
```

Comment:
```suggestion
WebEngine apps use the vanilla JavaScript build, and are set up in a similar fashion to those JS apps because it will also run in a browser. Set up the `index.js` and `index.html` file like in the **Vanilla JavaScript Setup** section. The majority of the configuration for the app will now be separated into a `manifest.js` file and then imported into the `index.html` file. Create a `manifest.js` file like below and save it in the same directory as the other two files. The entrypoint's value should have the same name as your app's HTML file. 
```